grpc-examples/routeguide/server/RouteGuide_sample_service.bal

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/grpc;
+import ballerina/time;
+
+listener grpc:Listener ep = new (8980);
+
+@grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR, descMap: getDescriptorMap()}
+service "RouteGuide" on ep {
+
+    isolated remote function GetFeature(Point point) returns Feature|error {
+        Feature?|error feature = featureFromPoint(point);
+        if feature is Feature || feature is error {
+            return feature;
+        } else {
+            return {location: {latitude: 0, longitude: 0}, name: ""};
+        }
```

Comment:
```suggestion
        foreach Feature feature in FEATURES {
            if feature.location == point {
                return feature;
            }
        }
        return {location: point, name: ""};
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/320/commits/7a57af9701dc7803c8d433a610e7e9f62404a75f